src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -783,7 +783,7 @@ public int CompareTo(String strB)
         }
 
         [Pure]
-        internal bool EndsWith(char value) {
+        public Boolean EndsWith(char value) {
```

Comment:
This could be simplified as:

```c#
public bool EndsWith(char value)
{
    int thisLen = this.Length;
    return thisLen != 0 && this[thisLen - 1] == value;
}
```